hypnotoad/gui/gui.py

```diff
@@ -35,6 +35,10 @@ class HypnotoadGui(QMainWindow, Ui_Hypnotoad):
 
     """
 
+    gui_options = options.Options(
+        grid_file="bout.grd.nc", plot_xlow=True, plot_ylow=True, plot_corners=True,
+    )
```

Comment:
These are not currently made available through the gui, although they would be if they were in a loaded yaml file.

Do we want them to be? Should they just be in with the general options, or in their own section?

Reply:
These can be in the yaml file https://github.com/boutproject/hypnotoad/blob/master/geqdsk_cdn.yaml#L45

Reply:
I was thinking we probably mostly just want them on, but we could have a menu item or something to set them, or a context menu on the plot widget or something, but didn't get around to implementing it. I don't particularly like the idea of an input file for interactive options that don't affect the final grid file, although that would probably be the easiest way to make it possible to change them if the user really wanted to, so could be worth adding.

Reply:
Something like under a `Settings > Preferences` menu? With an option to save/load preferences to `~/.config/hypnotoad/settings.yml` maybe?

Reply:
The way I've set up the gui here, the plotting options won't be used from the yaml file even if they're there (because they're not in `TokamakEquilibrium.default_options`. If we want to support having them in the yaml file in the gui too, I'd rather they were in a separate section, so it's clear they don't affect the generated grid.

Reply:
> Something like under a File > Preferences menu? With an option to save/load preferences to ~/.config/hypnotoad/settings.yml maybe?

That would be nice!